package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Editor for AsyncAPI documentation.",
   "scripts": {
     "start": "node src/server/index.js",
+    "dev": "node src/server/index.js",
```

Comment:
@derberg Added your suggestion but now `start` and `dev` look the same. Do you think it's still worth it to have `npm run dev`?

Reply:
I guess dev is not really needed